### DISCUSSION TITLE
arduino Lilygo_T_DISPLAY_S3 not working after update to tft_eSPI 2.5.0

### DISCUSSION THREAD

**Discussion by jkrijger67** [Bodmer/TFT_eSPI#2506] [General]
Hello,

I updated the Library for TFT_ESPI to 2.5.0. Since then 
Lilygo T_display_S3 isn't working anymore. I Tried to install 2.4.71 again
but this release isn't available anymore.

Help needed to bring back the display of lilygo T_Display_S3 again.
Where can I download tft_eSPI 2.4.71?

Johannes

**Comment by Bodmer** [comment]
Releases are here:
https://github.com/Bodmer/TFT_eSPI/releases

When the library is up dated the setup files must be edited again as they get replaced with defaults. See ReadMe.  Other users have the latest version working with that display so that is likely the cause.

**Comment by jkrijger67** [comment]
Hello Sir,
   
  
    
   
  
   Thank you for Information.
   
  
    
   
  
   I closed the request because after trying a few times I was able to run
   
  
   release 2.5.0.. So everything is fine.
   
  
    
   
  
   Thank you very much.
   
  
    
   
  
   Johannes
   
  
    
   
  
    
   
  
    
   
   
   
    Bodmer ***@***.***> hat am 23.03.2023 05:30 CET geschrieben:
    
   
     
    
   
     
    
     
   Reopened #2506.
